### ISSUE TITLE
[jobs/trucking] Three ideas for truckers

### ISSUE THREAD

**Issue by magustaliesin** [InZidiuZ/op-framework-issue-tracker#129]
**Name:**
XamotDB

**Summary:**
Three ideas that would make the trucker job more RP oriented. I like the profession, partly because driving a truck can be fun and challenging, but it's often solitary. But taking a job like towing, that relies on other players, can be very lonely when people just ditch broken cars, or fix them on their own. So...
1. Add a CB radio, either in game, or dedicated channel on discord. Lock it to channel 19, or open other channels for emergency calls, etc. We could talk to each other, call out where Smokies are set up, and so on.
2. Add branded trailers. I like that we have a cargo, but if you forget you have to make something up. It would also make the haul more interesting than just the standard rusted gray or blue trailer.
3. Add less than legal cargo options. Overweight loads, illegal goods, and so on could give the police a reason to set up weigh stations to check cargo. It would allow truckers who wanted to, to take the risk to earn a little more cash. Make those cargoes pay more, but obviously there's the risk of a search and fine/loss of license.

**Reason:**
I have found on many servers that outside of mechanic, EMS, and Police, the jobs just become something people do to grind for money. Then when they want to RP they all meet in the center of the city to kidnap each other or show off the cars they bought. This would give a little more interest to the trucker job and possibly encourage RP rather than grinding.

It doesn't have to be limited there either. Towing and cabbies could use a radio as well.

**Comment by InZidiuZ** [comment]
>Lock it to channel 19, or open other channels for emergency calls

What exactly do you mean by that?

>less than legal cargo options

I think this sounds dope and think it's something we could make happen. On the top of my head, there are two ways we could approach this. Either by adding some labeled jobs at the cargo stations, stating that they're illegal transports, by making their background behind that job red, for instance. OR, in the future, when we have an in-depth company system, we could let companies (registered and ran by players, of course) purchase different cargo stations, and then let them decide whether they would like to accept illegal jobs or not.

We would also have to give the ability for cops to search trailers, etc., but I think it's doable. A lot of other things would have to happen prior to this, of course, as we're still just over a week into this new framework. A lot of basic features, and more needed complex ones are needed before we can put this in to motion.

**InZidiuZ**
*OP-FW Developer*

**Comment by magustaliesin** [comment]
Appreciate the response.

What I mean by the first comment is, I'm not sure how much an actual CB radio system would be needed, or if you'd want to limit it just to those jobs that realistically use it. For the real citizen's band, channel 19 is used for truckers. There's an emergency channel as well but I forget which one that is. It may be too much of a mess to actually have 40 radio channels for everyone to use so it may just be easier to call it channel 19 for RP purposes, whether it be a chat channel in Discord we can just join and RP as if it were a radio channel, or implement some in game voice chat limited to the couple of jobs that use it and call it channel 19.
________________________________

**Comment by InZidiuZ** [comment]
Closing as outdated. If this is still relevant, please create a new issue.
